playgame.py

```diff
@@ -24,6 +25,11 @@ def main(argv):
     else:
         run_one_game(args)
 
+def ensure_path(path):
+    directories_from_path = "/".join(path.split("/")[:-1])
+    if os.path.isdir(directories_from_path) is False and directories_from_path != '':
+        os.makedirs(directories_from_path)
```

Comment:
If you take the `dir_name` changes, this will need to change to match
```suggestion
        os.makedirs(dir_name)
```

Reply:
I'm cool with it as is